terraform/aws/infrastructure.tf

```diff
@@ -236,3 +236,77 @@ resource "aws_security_group_rule" "grafana_server" {
 
   security_group_id = local.security_group_id
 }
+
+# IBSM imported resources
+
+data "aws_vpc" "ibsm" {
+  count = var.ibsm_vpc_id != "" && var.ibsm_tgw_project_tag != "" ? 1 : 0
```

Comment:
In the local section, somewhere after line 21, add something like
```
ibsm_count = var.ibsm_tgw_project_tag != "" ? 1 : 0
```

and then change
```
data "aws_vpc" "ibsm" {
  count = var.ibsm_vpc_id != "" && var.ibsm_tgw_project_tag != "" ? 1 : 0
  id    = var.ibsm_vpc_id
}
```

to

```
data "aws_vpc" "ibsm" {
  count = local.ibsm_count
  id    = var.ibsm_vpc_id
}
```